### DISCUSSION TITLE
Can't resolve images in css file

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#14858] [Help]
Hey everyone, I'm probably missing out on something fundamental.

How can I reference png images from a css file within a nextjs project?

I'm definitely referencing the file path correct.

I tried placing the images inside of the public folder.

and from the css file I reference: '/images/name-of-file.png', where images is the container folder nested inside of the public folder.

Can somebody point out what I'm doing wrong?

**Comment by sauhard98** [comment]
For referencing images from public folder you have to use the url tag. Read further at [https://developer.mozilla.org/en-US/docs/Web/CSS/url()](url)
Example: `background-image: url('images/my-image.png');`
